bundles/org.openhab.automation.pythonscripting/README.md

```diff
@@ -40,547 +40,238 @@ class Test:
         self.logger.info("Rule was triggered")
 ```
 
-or another one, using the [scope module](#module-scope)
+### `PY` Transformation
 
-```python
-from openhab import rule
-from openhab.triggers import ItemCommandTrigger
+Or as transformation inline script
 
-import scope
-
-@rule( triggers = [ ItemCommandTrigger("Item1", scope.ON) ] )
-class Test:
-    def execute(self, module, input):
-        self.logger.info("Rule was triggered")
+```text
+String Test "Test [PY(|'String has ' + str(len(input)) + 'characters'):%s]"
 ```
 
-::: tip Note
-By default, the scope, Registry and logger is automatically imported for UI based rules
-:::
-
-## `PY` Transformation
-
-openHAB provides several [data transformation services](https://www.openhab.org/addons/#transform) as well as the script transformations, that are available from the framework and need no additional installation.
-It allows transforming values using any of the available scripting languages, which means Python Scripting is supported as well.
-See the [transformation docs](https://openhab.org/docs/configuration/transformations.html#script-transformation) for more general information on the usage of script transformations.
-
-Use Python Scripting as script transformation by:
-
-1. Creating a script in the `$OPENHAB_CONF/transform` folder with the `.py` extension.
-   The script should take one argument `input` and return a value that supports `toString()` or `null`:
-
-   ```python
-   "String has " + str(len(input)) + " characters"
-   ```
-
-   or
+### More Scripting
 
-   ```python
-   def calc(input):
-       if input is None:
-           return 0
+A complete Documentation about Python Scripting Rules and Transformation Scripts can be found at
 
-       return "String has " + str(len(input)) + " characters"
-   calc(input)
-   ```
+[>> openHAB Python Scripting <<](https://github.com/openhab/openhab-python/blob/main/README.md)
```

Comment:
Would be nice to integrate this into the openHAB website where the other docs reside. We can take care of that later though, please remind me though.

Reply:
The idea was to split the binding documentation from the helper lib documentation.

Both documentations together was getting longer and longer. I still gave some scripting examples in the binding documentation itself, to get an impression how it works. But the whole helper lib documentation should resist in the same repo as the helper lib itself. Before, I was always synchronizing the README's after each tiny change in the helper libs, which affects the README.  Sometimes, I forgot to synchronize.